src/resources/views/crud/fields/relationship.blade.php

```diff
@@ -14,7 +14,7 @@
     // if field is not ajax but user wants to use InlineCreate
     // we make minimum_input_length = 0 so when user open we show the entries like a regular select
     $field['minimum_input_length'] = ($field['ajax'] !== true) ? 0 : ($field['minimum_input_length'] ?? 2);
-
+    if($field['relation_type'] == 'BelongsTo' || ($field['relation_type'] == 'BelongsToMany' && !isset($field['pivotFields']))) {
```

Comment:
`switch` statement please. Lots of options, so it'll be easier to read them one per line.

Reply:
I think here it's only two options, `1 || (2 && 3)` but only `1` and `2` compare the same thing.